### PR TITLE
Updates to `dplyr` prior to March workshop

### PR DESCRIPTION
**Pull request by ndrewwm**
@ellisvalentiner All that needs to happen next is knitting the book, right?

### CODE REVIEW

File: _bookdown.yml [hunk 1]
```diff
@@ -8,5 +8,6 @@ rmd_files: [
   "lessons/dataframes/dataframes.Rmd",
   "lessons/packages-and-rmarkdown/packages.Rmd",
   "lessons/packages-and-rmarkdown/rmarkdown.Rmd",
+  "lessons/dplyr/dplyr.Rmd"
```
Comment (ellisvalentiner) [inline _bookdown.yml]: missing a `,` at end of line

**Comment by ellisvalentiner** [comment]
@mooreaw the book will be automagically rebuild when the PR is merged into master (or possibly before if the tests pass…)

File: _bookdown.yml (hunk 1 above)
Comment (ndrewwm) [reply _bookdown.yml]: doh

**Comment by ellisvalentiner** [comment]
It looks like `dplyr` isn't installed. Can you sneak in a `install.packages(dplyr)` in there somewhere?

**Pull request merged**